src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -469,31 +436,43 @@ export class BaseEditCourseForm extends React.Component {
             />
             <Field
               name="tags"
-              itemType="topic"
-              component={RenderInputTextField}
-              fetchSuggestions={fetchCourseTagSuggestions(allCourseTags)}
-              renderSuggestion={renderCourseTagSuggestion}
-              type="text"
+              component={ReduxFormCreatableSelect}
               label={(
                 <>
                   <FieldLabel
                     id="tags.label"
                     text="Topics"
                     helpText={(
                       <p>
-                        You can add comma separated tags in the format mba-no-gmat,mba,mba_4_modules,mba_NY
+                        You can add tags in the format mba-no-gmat,mba,mba_4_modules,mba_NY
                       </p>
                     )}
                     optional
                   />
-                  {courseTags
-                    && courseTags.length ? this.formatCourseTags(courseTags, setCourseTags) : null}
                 </>
               )}
               disabled={disabled || !administrator}
               value={this.state.courseTag}
-              onChange={(e) => this.handleCourseTagChange(e, courseTags, setCourseTags)}
               optional
+              currValue={
+                Array.isArray(courseTags)
+                  ? courseTags.map(tag => ({
+                    label: tag,
+                    value: tag,
+                  }))
+                  : null
+              }
```

Comment:
Keep this as a util function, helps with readability.

Reply:
done